_submissions/round-12/3/2015-3-28-andrew-macdonald.md

```diff
@@ -0,0 +1,42 @@
+---
+data: '2015-03-28'
+round: Round 12
+title: Demotivation
+author: Andrew MacDonald
+permalink: 2015/3/andrew-macdonald-motivation
+tags: Motivation
+
+---
+
+
+### Part 1
+
+When I was an undergrad in my one-and-only computer course (it was on Visual Basic, whatever that is) the instructor clearly didn’t care at all about the lesson or our understanding of it. He taught directly from the book without varying one bit , and never related the material to any practical application. His indifference to the material gave me the feeling that this was boring, unnecessary stuff -- and it was years until I went back to programming of any kind.
+
+It’s important to relate what you’re teaching to the student’s motivations. For example, choosing examples that are related to what students are doing is exciting -- but choosing something abstract just because it is convenient for instruction can seem boring and esoteric.  Choosing relevant examples also shows how you care about what your students are doing, too.
+
+### Part 2
```

Comment:
This is really good stuff - can you please submit it as a separate post so that it doesn't get lost?  (Having it as the second half of your post will probably cut readership in half or more.)
Thanks for writing it...